src/applications/benefits-optimization-aquia/21-0779-nursing-home-information/config/transform.js

```diff
@@ -16,18 +16,39 @@ export const transform = (formConfig, form) => {
     medicaidStartDate,
     monthlyCosts,
     nursingOfficialInformation,
+    signature,
   } = form?.data;
 
   const { nursingHomeName, nursingHomeAddress } = nursingHomeDetails || {};
   const nursingOfficialName = `${nursingOfficialInformation?.fullName?.first?.trim() ||
     ''} ${nursingOfficialInformation?.fullName?.last?.trim() || ''}`.trim();
 
-  // No claimant info if veteran is the patient
+  // Convert country code from 3-letter to 2-letter format
+  const getCountryCode = country => {
```

Comment:
I pushed a change to allow country of length 2 or 3, so you can ditch this conversion (now or later)